TESTING.md

```diff
@@ -342,6 +342,8 @@ While running tests, the following environment variables can be used:
 * `VCD_TOKEN` : specifies the authorization token to use instead of username/password
    (Use `./scripts/get_token.sh` to retrieve one)
 * `GOVCD_KEEP_TEST_OBJECTS` will skip deletion of objects created during tests.
+* `GOVCD_API_VERSION` allows to select the API version to use. This must be used for testing purposes only as the SDK
```

Comment:
Maybe let's highlight "for testing purposes only" just in case.

Reply:
Good point! Highlighted already 😄